code/game/machinery/cryopod.dm

```diff
@@ -720,5 +719,6 @@
 			O.loc = R
 		qdel(I)
 	qdel(R.module)
+	qdel(occupant)
```

Comment:
These changes should not be necessary - the borg's built-in MMI is deleted by the code above, and the mob itself is deleted in the parent call of `despawn_occupant`.

It seems the actual issue is that the cyborg cryo system lacks a control computer, and thus does not automatically despawn the cyborg when they ghost - allowing them to re-enter, since ghosting inside a cryopod allows body re-entrance, even if the cryopod doesn't actually despawn the person.